src/main/java/uk/co/onsdigital/discovery/model/HierarchyEntry.java

```diff
@@ -42,6 +51,8 @@
     @ManyToOne
     @JoinColumn(name = "hierarchy_id", nullable = false)
     private Hierarchy hierarchy;
+    @Column(name="hierarchy_id",updatable = false, insertable = false)
+    private String hierarchyId;
```

Comment:
Does this need to be mapped twice? Can't you always just do `getHierarchy().getId()`? (Or add a `.getHierarchyId()` method that does that for you).